src/Sql/Sql/help/Get-AzSqlDatabaseLongTermRetentionBackup.md

```diff
@@ -67,43 +67,52 @@ The **Get-AzSqlDatabaseLongTermRetentionBackup** cmdlet gets all long term reten
 PS C:\> Get-AzSqlDatabaseLongTermRetentionBackup -Location northeurope
 
 
-BackupExpirationTime : 3/22/2018 5:50:55 AM
-BackupName           : 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
-BackupTime           : 3/15/2018 5:50:55 AM
-DatabaseName         : database01
-DatabaseDeletionTime :
-Location         : northeurope
-ResourceId           : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/resourceGroups/resourcegroup01/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server01/longTermRetentionDatabases/database01/longTermRetentionBackups/601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
-ServerName           : server01
-ServerCreateTime     : 2/29/2018 12:12:19 AM
-
-BackupExpirationTime : 3/22/2018 11:43:18 PM
-BackupName           : 55970792-164c-4a4a-88e5-7158d092d503;131656309980000000
-BackupTime           : 3/15/2018 11:43:18 PM
-DatabaseName         : database02
-DatabaseDeletionTime : 3/18/2018 4:36:00 PM
-Location         : northeurope
-ResourceId           : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server02/longTermRetentionDatabases/database02/longTermRetentionBackups/55970792-164c-4a4a-88e5-7158d092d503;131656309980000000
-ServerName           : server02
-ServerCreateTime     : 2/28/2018 12:12:19 AM
-```
-
-This command gets all long term retention backups for all databases (which may be alive or deleted) in northeurope, resource group will be set only if server is live.
+BackupExpirationTime			 : 3/22/2018 5:50:55 AM
+BackupName						 : 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+BackupTime						 : 3/15/2018 5:50:55 AM
+DatabaseName					 : database01
+DatabaseDeletionTime			 :
+Location						 : northeurope
+ResourceId						 : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/resourceGroups/resourcegroup01/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server01/longTermRetentionDatabases/database01/longTermRetentionBackups/601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+ServerName						 : server01
+ServerCreateTime				 : 2/29/2018 12:12:19 AM
+ResourceGroupName                : 
+CurrentBackupStorageRedundancy   : Geo
+RequestedBackupStorageRedundancy :
```

Comment:
why are we returning 'RequestedBackupStorageRedundancy'? it should be removed from GET call

Reply:
This is a property of the AzureSqlDatabaesLongTermRetentionBackupModel.cs since it's a required field for the SDK; therefore, because that type is what's outputed by GET, it is returned even if it's always null.

Reply:
As discussed offline, removing Requested* from GET ltr backup response altogether